src/ui/pages/RevealPrivateKey/index.ts

```diff
@@ -0,0 +1 @@
+export { RevealPrivateKey } from './RevealPrivateKey';
```

Comment:
Currently, this page is dedicated solely to revealing the private key, hence the new name